fastly_compute/src/fetcher.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use async_trait::async_trait;
+use fastly::{
+    Request as FastlyRequest,
+    Response as FastlyResponse,
+};
+use sxg_rs::http::{
+    HttpRequest,
+    HttpResponse,
+    Method,
+};
+
+pub struct FastlyFetcher {
```

Comment:
Could you add comments documenting these structs/methods? Thanks.

Reply:
Done